output/slide-deck.qmd

```diff
@@ -342,186 +299,127 @@ Nesting repositories can give problems
 :::
 
 ::: aside
-\*: You will only be asked once
+\*: You will only be asked once (by computer)
 :::
 
 ::: notes
-9: Signing-in: Necessary because the repository is "private"
-:::
-
-## Clone repository (and IV) {.smaller}
-
-<br>
-
-Try the following:
-
-::: nonincremental
--   Clone someone else's repo (\*)
-
-OR
-
--   Clone <https://github.com/emmapett/VC_Workshop> (\*)
-:::
-
-::: fragment
-::: callout-important
-If you get
-
-``` text
-fatal: destination path 'my-first-repo' already exists and is not an empty directory.
-```
+6: Note that Rstudio fills in the folder name for you automatically; you don't
+need to create a new folder to "nest" your repository into it, it will create it
+automatically. If it already exists and is not empty, it will give you an error
+and won't create the repository. In that case, you can input another folder name
+of your choice.
 
-try a different local folder name
-:::
-:::
+7:
 
-::: aside
-\*: Select "Open in new session" this time
-:::
+Avoid using a network or cloud storage folder (e.g. Dropbox, OneDrive); they can
+produce synchronization conflicts
 
-::: notes
--   Let Emma show how she organizes her repository and what is its usefulness
+Network units can conflict with the R environment (more about this in a
+while...)
 
--   You are not asked again to authenticate!!
+**NEVER** clone a repository inside another one
 
--   The folder names must not "collapse", but the local folder name does not
-    need to be the same as the name of the repository. If you are so unlucky as
-    to have a repo with the same name as someone else's, try adding e.g. the
-    other user's name as suffix.
+9: Signing-in: Necessary because the repository is "private"
 :::
 
 ## Explore your local repo
 
--   `.git`: Home of your git repository
+-   ".git": Home of your git repository
 
 -   Folders
 
 ::: {.fragment .nonincremental}
 -   Files:
-    -   `.gitignore`
+    -   ".gitignore"
 
-    -   `README.md` and `README.Rmd`
+    -   "README.md" and "README.Rmd"
```

Comment:
This one probably doesn't need an extra description

Reply:
I clarified better is use in the speaker notes. I think it's good to distinguish between the template "README.md", which will be by default in the project, and the one they will need to create for the project (we will do this later on by knitting "README.Rmd", so I dropped the reference to that in this slide).

How does it look now? 🙂